app/Http/Controllers/SongController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Song;
+use Illuminate\Http\Request;
+
+class SongController extends Controller
+{
+    public function index()
+    {
+        return Song::all();
```

Comment:
Is this meant to be an API route? Shouldn't we have a view file that presents the results?